src/routes/games.js

```diff
@@ -59,15 +86,13 @@ module.exports = db => {
       })
   });
 
-  // getAllGamesByUserIDNEventID
-  // router.get("/event/:eventID/games/testing", (req, res) => {
-  //   const eventID = req.params.eventID;
-  //   const userId = getLoggedUserId(req);
-  //   getAllGamesByEventID(db, userId, eventID)
-  //     .then(data => {
-  //       res.json( {getAllGamesByEventID: data} );
-  //     })
-  // });
-
+  router.get("/event/:eventID/games/:userID/", (req, res) => {
+    const eventID = req.params.eventID;
+    const userID = req.params.userID;
+    getAllGamesForPlayerInEvent(db, userId, eventID)
+      .then(data => {
+        res.json( {getAllGamesByEventID: data} );
```

Comment:
getAllGamesByEventID looks like a function name, and here it is just a "game". I suggest you to just change the key to "game"